src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -135,15 +131,8 @@ const InfiniteScroll = ({
         );
       }
     };
-  }, [
-    beginPage,
-    endPage,
-    lastPage,
-    multiColumn,
-    pageArea,
-    pageHeight,
-    replace,
-  ]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [beginPage, lastPage, multiColumn, pageArea, pageHeight, replace]);
```

Comment:
endPage in the dependency array caused this effect to be called an additional time and called onMore() an additional time.